src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -234,6 +234,11 @@ private void LogInitialization(IHost host, bool isOffline, int attemptCount, int
             var log = isOffline ? "Host is offline." : "Initializing Host.";
             logger.LogInformation(log);
             logger.LogInformation($"Host initialization: ConsecutiveErrors={attemptCount}, StartupCount={_hostStartCount++}");
+
+            if (_scriptWebHostEnvironment.InStandbyMode)
+            {
+                logger.LogInformation("Host is in standby mode");
```

Comment:
This belongs either in the `StandbyInitializationService` or the `StandByManager`. I would go with the service. This service doesn't need to be aware of standby logic.

Reply:
We don't have a hook into host startup in either of those places, so we don't have a place to log any details about the currently-starting host (those are WebHost services). Thoughts on how I could log that in a useful way from either of those places?

Reply:
We do have the System logger at the WebHost level (which will not be deferred) -- but my point was: if the host restarts a few times, we don't even know about it to log "This host is in standby mode". We don't fire an event or anything when that happens. In the scope of those services, we have no knowledge of a host starting up.